ports/at91sam7s/CMakeLists.txt

```diff
@@ -122,6 +122,7 @@ set(BACNET_PROJECT_SOURCE
     ${LIBRARY_BACNET_BASIC}/service/h_dcc.c
     ${LIBRARY_BACNET_BASIC}/service/h_apdu.c
     ${LIBRARY_BACNET_BASIC}/npdu/h_npdu.c
+	${LIBRARY_BACNET_BASIC}/npdu/s_router.c
```

Comment:
it will save me time in the future if these lines can be aligned

Reply:
Removed. But I promise to remember.